plugins/woocommerce-blocks/assets/js/blocks/checkout/styles/editor.scss

```diff
@@ -8,6 +8,10 @@
 		align-items: flex-start;
 		padding-left: 5px;
 
+		&:has(> :first-child.wp-block-woocommerce-checkout-totals-block) {
```

Comment:
Same as [this comment](https://github.com/woocommerce/woocommerce/pull/49649/files#r1683059922).